project/cfmc30/README.rst

```diff
@@ -0,0 +1,124 @@
+FDCT: Fast Discrete Cosine Transformation Library
+=================
+
+Basic Information
```

Comment:
Please supply the github repository for the project.

Reply:
Please do not resolve comments.  Resolving a comment makes it hard to track.  Leave them open.